docs/reference/authorization.md

```diff
@@ -52,8 +52,12 @@ When a user makes a request to perform that action, the following process takes
 Each policy attached to a user or a group has an `Effect` - either `allow` or `deny`.
 During evaluation of a request, `deny` would take precedence over any other `allow` policy.
 
-This helps us compose policies together. For example, we could attach a very permissive policy to a user and use `deny` rules to then selectively restrict what that user can do.
+The default if no permissions exist is to allow nothing.
 
+This helps us compose policies together. For example: 
+
+* we could attach a very permissive policy to a user and use `deny` rules to then selectively restrict what that user can do.
+* Since `deny` is stronger than `allow`, you could specify `fs:*` `allow` with `fs:Write*` `deny` to a user with the net effect that the user will be able to do all fs commands _except_ write commands.
```

Comment:
I would strongly like to discourage doing this.  For instance, `fs:Create*` and `fs:Delete*` will still be allowed using this configuration, including `fs:CreateMetaRange`.  So if you gave me this policy, I could probably do almost everything you think you are preventing.

I would prefer to give an example of denying by path.  So you could allow me to do some things on a repo, but prevent me from doing anything under some prefix.